contracts/test/Assessment/AssessmentUtilsLibTest.sol

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: GPL-3.0-only
+
+pragma solidity ^0.8.0;
+
+import "../../interfaces/IAssessment.sol";
+import "../../libraries/Assessment/AssessmentUtilsLib.sol";
+import "hardhat/console.sol";
+
+/// Used as a helper to test internal pure and view functions of AssessmentUtilsLibTest
+contract AssessmentUtilsLibTest {
+
+  function abs(int x) external pure returns (int) {
+    return AssessmentUtilsLib.abs(x);
+  }
+
+  function min(uint a, uint b) external pure returns (uint) {
+    return AssessmentUtilsLib.min(a, b);
+  }
+
+  function pollFraudExists(IAssessment.Poll memory poll) external pure returns (bool) {
+    return AssessmentUtilsLib.pollFraudExists(poll);
+  }
+
+  function _getPollStatus(IAssessment.Poll memory poll) external view returns (IAssessment.PollStatus) {
+    return AssessmentUtilsLib._getPollStatus(poll);
+  }
+
+  function _getPayoutImpactOfClaim (IAssessment.Claim memory claim) external pure returns (uint) {
+    return AssessmentUtilsLib._getPayoutImpactOfClaim(claim);
+  }
+
+  function _getPayoutImpactOfIncident (IAssessment.Incident memory incident) external pure returns (uint) {
+    return AssessmentUtilsLib._getPayoutImpactOfIncident(incident);
+  }
+
+  function _getVoteLockupEndDate (
+    IAssessment.Configuration calldata CONFIG,
+    IAssessment.Vote memory vote
+   ) external pure returns (uint) {
+    return AssessmentUtilsLib._getVoteLockupEndDate(CONFIG, vote);
+  }
+
+  function _getCooldownEndDate (
+    IAssessment.Configuration calldata CONFIG,
+    uint32 pollEnd
+  ) external pure returns (uint32) {
+    return AssessmentUtilsLib._getCooldownEndDate(CONFIG, pollEnd);
+  }
+
+  function _calculatePollEndDate (
```

Comment:
Do not use a separate function to unpack the variables. Define them in-place (if really needed), ex: `uint96 accepted = poll.accepted;`

if the calls are removed from `submitClaim` and `submitIncident`, we can move this function to `AssessmentVoteLib`